app/src/main/java/org/kiwix/kiwixmobile/zimManager/ZimManageViewModel.kt

```diff
@@ -150,7 +150,7 @@ class ZimManageViewModel @Inject constructor(
       updateLibraryItems(booksFromDao, downloads, networkLibrary, languages),
       updateLanguagesInDao(networkLibrary, languages),
       updateNetworkStates(),
-      requestsAndConnectivtyChangesToLibraryRequests(networkLibrary),
+      requestsAndConnectivityChangesToLibraryRequests(networkLibrary),
```

Comment:
Please revert this. This is not related to the ticket.

Reply:
hi @gouri-panda ,

done